oci/client.py

```diff
@@ -814,6 +814,7 @@ def delete_manifest(
         image_reference: om.OciImageReference | str,
         purge: bool=False,
         accept: str=om.MimeTypes.prefer_multiarch,
+        absent_ok: bool=True,
```

Comment:
I think we should default to `False` (or adjust all current usages) to ensure no break in current behaviours.